### ISSUE TITLE
removeTextPart() issue?

### ISSUE THREAD

**Issue by postme** [zbateson/MailMimeParser#22]
Hi Zaahid,

I have a text/plain message where I retrieve the text content with getTextContent(). I convert this to a HTML representation and add this to the message via setHtmlPart(). I subsequently remove the text part via removeTextPart(). This last action causes duplication of the HTML content, ie the same HTML message is now present twice in the same mime part. This doesn't happen when the text/plain message has an attachment. 

```
$message = parseInput(EMAIL);
$text = $message->getTextContent();
$body = '<pre>' . $text . '</pre>';
$content = renderTemplate(TEMPLATE_OUTGOING, ['body' => $body, 'tracker_url' => $trackerUrl]);
$message->setHtmlPart($content, 'utf8');
$message->removeTextPart();
```

Calling $message->removeTextPart(); earlier in the process doesn't yield any effect, the text part remains in the e-mail body.

Kind regards
Meint

**Comment by zbateson** [comment]
Thanks Meint, I'll take a look :smile:

**Comment by zbateson** [comment]
Hi Meint,

Fixed in the just released 0.3.2 :) please reopen/let me know if you have any issues.

Zaahid

**Comment by postme** [comment]
Hi Zaahid, works correctly, really happy with this fix because I couldn't go live with my solution without this :-)
